### ISSUE TITLE
Null input to trim() deprecated on PHP 8.1

### ISSUE THREAD

**Issue by neyre** [gabordemooij/redbean#895]
Getting the following error when doing anything on PHP 8.1.4, redbean version 5.7.1.

```
Internal Server Error
trim(): Passing null to parameter #1 ($string) of type string is deprecated
```

I was able to fix it by replacing:
`if ( trim( $sql ) === '' ) {` with: `if ( is_null($sql) || trim( $sql ) === '' ) {`
in the main `glueSQLCondition` function within the main redbean php file, line 6191.

Fix credit from closed issue here: https://github.com/gabordemooij/redbean/issues/871#issuecomment-1042222561

**Comment by gabordemooij** [comment]
Yep, another 8.1 compatibility issue probably, I will look into this. Thanks for reporting.

**Comment by gabordemooij** [comment]
Seems to have already been fixed in 5.7.2?

**Comment by neyre** [comment]
Yep, looks fixed in 5.7.2. Thanks!
